src/Chirp.Web/Pages/Public.cshtml

```diff
@@ -38,20 +38,23 @@
                         <div>
                             <strong><a href="/@cheep.Author">@cheep.Author</a></strong>
                             <small>&mdash; @cheep.Timestamp</small>
-                            <!-- YouTube Link Check and Embed -->
+                            <!-- YouTube Link Check and Embed, hashtags -->
                             @{
                                 string Message;
                                 var embedUrl = Model.GetYouTubeEmbed(cheep.Message, out Message);
+
+                                List<string> hashTags;
+                                string MessageWithHashtags;
+                                hashTags = Model.GetHashTags(Message, out MessageWithHashtags);
                                 
                                 if (embedUrl != null)
                                 {
                                     <iframe width="560" height="315" src="@embedUrl" frameborder="0" allow="accelerometer; autoplay; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
                                 }
 
-                                <p>@Message</p>
+                                <p>@Html.Raw(MessageWithHashtags)</p>
```

Comment:
Is html.raw not one of the things that we should be careful of because of possible injections? Are we ensuring this isn't a security leak?

Reply:
This is indeed a security risk, ill fix it.

Reply:
this is now fixed to be better structured without html.raw running